src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/ChangeContactDetailsCallbackHandler.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.caseworker;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.ccd.domain.CaseEvent;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.Role;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.Callback;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.CallbackHandler;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.CallbackType;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.CHANGE_CONTACT_DETAILS;
+import static uk.gov.hmcts.cmc.claimstore.services.ccd.Role.CASEWORKER;
+
+@Service
```

Comment:
Please put this behind feature toggle

Reply:
I am just getting confirmation from Brian whether this is needed or not. The ticket is not clear

Reply:
put up the feature toggle in it